src/bindings/interface/importsource.i

```diff
@@ -2,6 +2,8 @@
 
 #define LIBCELLML_EXPORT
 
+%include "createconstructor.i"
+
```

Comment:
Why the empty line? Also, why is `entity.i` included before `types.i`? Kind of the same for `model.i` below (and elsewhere too).